README.md

```diff
@@ -316,6 +321,13 @@ The migration file is a normal Bash script file:
 - It should be idempotent as much as possible to ensure that the changes are
   not duplicated to the pipeline when run the migration multiple times.
 - Pass the `shellcheck` without customizing the default rules.
+- Check whether the migration is for all kinds of Konflux pipelines or not. If
+  no, skip the pipeline properly in the script, e.g. skip FBC pipeline due
+  to [many tasks are removed](https://github.com/konflux-ci/build-definitions/blob/main/pipelines/fbc-builder/patch.yaml)
+  from template-build.yaml.
+- The pipeline file path and name can be arbitrary. Please do not use the input
```

Comment:
What is input value? Why a user shouldn't do it?

Reply:
In build-definitions pr 2018, the author matches pipeline file to determine pipeline types for conditional operation.

It can ends with a user-defined extracted pipeline file name or be a temporary file. It's the implementation details. So, as mentioned in the doc, from user's point of view, the pipeline file value can be arbitrary and should not be relied on to match anything.

Reply:
I think the explanation from your second paragraph should be included into the doc.

Reply:
I don't think so. It's implementation details which can be changed by developers at any time when need. It should not be exposed to users, if yes, some migrations may try to match file path according to this document.